deploy/build-plugin.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo 'Starting building plugin'
+echo '*****************'
+
+# Clone Kibana source code in "kibana" folder
+# git clone https://github.com/elastic/kibana.git kibana --progress
+
+echo 'Kibana source code is already downloaded from Docker, for caching reasons'
+
+# Kibana version for CUTLER project is pre-agreed to 7.4.0
+# Change to branch 7.4 which represents Kibana version 7.4.0
+cd kibana && git checkout 7.4
+
+# Create required plugins directory where we will place our plugin
+# through symbolic link
+mkdir plugins/ && mkdir plugins/Draxis-map-layers-kibana-plugin
```

Comment:
using -p, you can create a folder and parent folders

`mkdir -p plugins/Draxis-map-layers-kibana-plugin`